lv_port/lv_port_display_espressif.c

```diff
@@ -0,0 +1,41 @@
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+
+#include "display_port.h"
+
+#include "sdkconfig.h"
+#include "driver/gpio.h"
+
+void display_port_delay(lv_disp_drv_t *drv, uint32_t delay_ms)
+{
+    (void) drv;
+
+    vTaskDelay(pdMS_TO_TICKS(delay_ms));
+}
+
+void display_port_backlight(lv_disp_drv_t *drv, uint8_t state)
+{
+    (void) drv;
+
+#ifdef CONFIG_LV_DISP_PIN_BCKL
+    gpio_set_level(CONFIG_LV_DISP_PIN_BCKL, state);
+#endif
+}
```

Comment:
I wanted to use `esp_lcd_backlight.c` for this feature, but we can leave it as it is

Reply:
We can use the backlight feature.

Reply:
I've created this initial patch, but I think we should discuss a bit further on how to use `esp_lcd_backlight`.

```diff
diff --git a/lv_port/lv_port_display_espressif.c b/lv_port/lv_port_display_espressif.c
index 9f6f4b1..4ecb218 100644
--- a/lv_port/lv_port_display_espressif.c
+++ b/lv_port/lv_port_display_espressif.c
@@ -13,13 +13,11 @@ void display_port_delay(lv_disp_drv_t *drv, uint32_t delay_ms)
     vTaskDelay(pdMS_TO_TICKS(delay_ms));
 }
 
-void display_port_backlight(lv_disp_drv_t *drv, uint8_t state)
+void display_port_backlight(lv_disp_drv_t *drv, disp_backlight_h *bckl, uint8_t brightness)
 {
     (void) drv;
-
-#ifdef CONFIG_LV_DISP_PIN_BCKL
-    gpio_set_level(CONFIG_LV_DISP_PIN_BCKL, state);
-#endif
+    
+    disp_backlight_set(bckl, brightness);
 }
 
 void display_port_gpio_dc(lv_disp_drv_t *drv, uint8_t state)
diff --git a/lvgl_tft/display_port.h b/lvgl_tft/display_port.h
index 2136ec6..4150e22 100644
--- a/lvgl_tft/display_port.h
+++ b/lvgl_tft/display_port.h
@@ -12,6 +12,8 @@ extern "C"
 #include "lvgl/lvgl.h"
 #endif
 
+#include "esp_lcd_backlight.h"
+
 /**
  * Busy wait delay port
  *
@@ -24,9 +26,10 @@ void display_port_delay(lv_disp_drv_t *drv, uint32_t delay_ms);
  * Backlight control port
  *
  * @param drv Pointer to driver See @ref lv_disp_drv_t
- * @param state State of the backlight signal
+ * @param bckl Pointer to backlight handler See @ref disp_backlight_h
+ * @param brightness Brightness percentage
  */
-void display_port_backlight(lv_disp_drv_t *drv, uint8_t state);
+void display_port_backlight(lv_disp_drv_t *drv, disp_backlight_h *backlight, uint8_t brightness);
 
 /**
  * DC signal control port

```